api/oauth/post.ts

```diff
@@ -62,7 +62,7 @@ const logic = async (
         ),
       }).apps
         .getInstallation({
-          installation_id: Number(customParams.data.installation_id),
+          installation_id: Number(customParams.data?.installation_id),
```

Comment:
How does this API know which installation to get? Presumably, we need to look it up from our API right?

Reply:
Good question.  They handle that in the backend?  We send the `client_id` of the app to their auth page.

https://github.com/RoamJS/query-builder/blob/main/src/components/ExportGithub.tsx#L181

Reply:
probably relevant: https://github.blog/changelog/2024-05-01-github-apps-can-now-use-the-client-id-to-fetch-installation-tokens/

Reply:
If it is not required for this GitHub flow, would it be alright to leave it as an empty string? 

I've tested it a few times and it seems to work like a charm

Reply:
yea that seems fine for now